src/custom/components/ClickWrap/index.tsx

```diff
@@ -0,0 +1,131 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink } from 'react-router-dom'
+import { ButtonPrimary, ButtonSecondary } from 'components/Button'
+
+const Wrapper = styled.div`
+  display: flex;
+  width: 100%;
+  height: auto;
+  background: ${({ theme }) => theme.bg1};
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 999;
+  min-height: 100px;
+  padding: 16px;
+  box-shadow: rgb(0 0 0) 0 -4px 0 0, rgb(0 0 0 / 16%) 0 -9px 0 0;
+  box-sizing: border-box;
```

Comment:
do you need this? we have 
```
* {
  box-sizing: border-box;
}
```
in the fixed global styles

Reply:
Second nature to add it :) - see it in the global styles now. I will enhance it further with:
> *, *:after, *:before { box-sizing:border-box; }

In our global styles and test it a bit.

Reply:
don't do it here

Reply:
Do it where? I added it to `src/custom/theme/baseTheme.tsx`